### PR TITLE
[wip] fixing and refactoring ovirt infra pages

### PR DESCRIPTION
**Pull request by eedri**
- lots of work and TBD, but starting to do some order

Signed-off-by: Eyal Edri <eedri@redhat.com>

Fixes issue # (delete if not relevant)

Changes proposed in this pull request:

-

-

-

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): (please @mention yourself to sign)

This pull request needs review by: (please @mention the reviewer if relevant)

### CODE REVIEW

**Comment by eedri** [comment]
@jekader please continue to update when possible, there is a lot of old files under the infra folder and some already are available on RTD, so they can be removed.

Whatever isn't on RTD, needs to be added, like the new networking PHX layout.

File: source/develop/infra/infrastructure-team-administrators.html.md [hunk 1]
```diff
@@ -8,62 +8,18 @@ authors: bproffitt, dcaroest, dneary, eedri, knesenko, quaid, rydekull
...
 
-*   Jenkins - jenkins.ovirt.org: eedri, dcaro, knesenko
-*   Gerrit - gerrit.ovirt.org: iheim, rydekull, dcaro
-*   Mailman - lists.ovirt.org: quaid, misc
-*   MediaWiki - www.ovirt.org: quaid, DNeary, garrett, misc
-*   OpenShift - www.ovirt.org: quaid, DNeary, garrett, rydekull
-*   Resources - resources.ovirt.org: misc, eedri, quaid, rydekull, dcaro
+*   Gerrit  - gerrit.ovirt.org: Itamar Heim (iheim@redhat.com), mostly for AWS management
+*   Mailman - lists.ovirt.org: Duck (duck@redhat.com)
+*   MediaWiki - www.ovirt.org: Duck (duck@redhat.com)
```
Comment (duck-rh) [inline source/develop/infra/infrastructure-team-administrators.html.md]: There is no wiki anymore, it died one year before I came

File: source/develop/infra/infrastructure-team-administrators.html.md [hunk 2]
```diff
@@ -8,62 +8,18 @@ authors: bproffitt, dcaroest, dneary, eedri, knesenko, quaid, rydekull
...
+*   OpenShift - www.ovirt.org: Duck (duck@redhat.com)
```
Comment (duck-rh) [inline source/develop/infra/infrastructure-team-administrators.html.md]: This is not on OpenShift at the moment, but on a VM at PHX

File: source/develop/infra/infrastructure.html.md [hunk 3]
```diff
@@ -17,35 +17,22 @@ The oVirt Infra Team is a volunteer effort to provide community infrastructure s
...
-*   **Gerrit Admins:**Eyal Edri (eedri), David Caro (Dcaroest), Anton Marchukov (amarchuk)
-*   **Puppet and Foreman Lead:** Ewoud Kohl van Wijngaarden (Ekohl), David Caro (Dcaroest),Barak Korren (Bkorren)
-*   **Mail Servers** Karsten Wade (Quaid),Michael Scherer (Misc)
+*   **Infrastructure Lead:** Evgheni Dereveanchin (ederevea)
+*   **Release Manager:** Sandro Bonazzola (sbonazzo)
+*   **CI Leads:** Barak Korren (bkorren)
+*   **Repositories/Build Lead:** Sandro Bonazzola (sbonazzo) 
+*   **Gerrit Admins:** Evgheni Dereveanchin (ederevea)
+*   **Puppet and Foreman Lead:** Barak Korren (Bkorren), Evgheni Dereveanchin (ederevea)
+*   **Mail Servers** Duck (duck)
```
Comment (duck-rh) [inline source/develop/infra/infrastructure.html.md]: Mail and Web

File: source/develop/infra/infrastructure.html.md [hunk 4]
```diff
@@ -96,18 +83,8 @@ The main thing is, come [communicate with us](#Communication) if you have any qu
...
-
-### oVirt Instances
-
-We have some oVirt instances installed and we use them for our infrastructure. For more information please click [here](/develop/infra/infrastructure-instances/)
+TBD
 
 ### Documentation
 
-We are in the process of moving all our documentaion to be inside our git repos, and published as HTML. You can find the published docs at: <http://ovirt-infra-docs.readthedocs.org/en/latest/>
+Most of what we do is documented in our own RTD page. You can find the published docs at: <http://ovirt-infra-docs.readthedocs.org/en/latest/>
```
Comment (duck-rh) [inline source/develop/infra/infrastructure.html.md]: we should add direction to contribute to the RTD code

**Comment by eedri** [comment]
Since this is going to be a very big refactor, might worth to split it to separate patches to address each file separately.

**Comment by ifireball** [comment]
I think the page structure where we try to put admins on everything is wrong and bound to get out of date quickly.

Instead, lets just have the following content on this page:
1. Key high-level responsibilities of the team (Public infra, mailing lists, CI system, etc.)
2. How to contact the infra team, (Jira, IRC, ML, etc.)
3. Who is in the team
4. How to join and help out

**Comment by gregsheremeta** [comment]
> The page need to be completely rewritten IMO, not just updated.

closing per that

**Pull request closed without merging**